### ISSUE TITLE
ApiTransactionService less verbose logging

### ISSUE THREAD

**Issue by oliverse-da** [digital-asset/daml#6916]
Introduced in Canton by @matthiasS-da namely:

- log prefix of first 20 characters identifiers: to `logging.parties(request.requestingParties` added `.map(_.take(20)))`
- log request `SimpleName` instead of entire request

**Comment by stefanobaghino-da** [comment]
I'm not convinced we should make this the hard-coded behavior.

**Comment by matthiasS-da** [comment]
I agree, it should not be hard-coded like this. This was a temporary fix to avoid very long lines in our log files.

My concerns with the current state:
- The request may contain confidential data. Therefore, there should be a configuration option to not log it.
- Typically, the request already contains the requesting parties. By adding them to the log context, they get logged twice.
- In Canton, party names are very long, because they are suffixed with a namespace hash. The log gets way more readable, if only a prefix is logged.
- Canton already has its own interceptor for logging requests / responses at GRPC interfaces. The ledger API server logging does not add value for us.

**Comment by oliverse-da** [comment]
Edited title and description above to better mirror the concerns listed by @matthiasS-da .

**Comment by daravep** [comment]
Stale issue and fixed via https://github.com/DACH-NY/canton/pull/33956
